src/every_query/eval_composite.py

```
@@ -0,0 +1,107 @@
+        task_labels_dir = str(task_set_dir / slug)
+
+        if not Path(task_labels_dir).is_dir():
+            logger.warning(f"Missing task_labels_dir for code={code}: {task_labels_dir} (skipping)")
+            continue
+
+        # Point datamodule at this code’s task dfs
+        train_cfg.datamodule.config.task_labels_dir = task_labels_dir
+        D = instantiate(train_cfg.datamodule)
+
+        out = trainer.test(model=M, datamodule=D, ckpt_path=cfg.ckpt_path)
+        m = out[0] if out else {}
```

Ruff (0.14.10)

Local variable `m` is assigned to but never used

Remove assignment to unused variable `m`

(F841)